src/tool.js

```diff
@@ -0,0 +1,39 @@
+exports.genForm = (inputs) => {
+    // Create a blank string to start with
+    var formStr = '';
```

Comment:
You could change `var` to `let` or `const` just to keep the consistency of your code!

Reply:
Good call. Changed `var` to `let` to maintain consistency and avoid potential global var collisions.